skimage/measure/_regionprops_utils.py

```diff
@@ -184,9 +184,6 @@ def euler_number(image, connectivity=None):
         return int(0.125 * coefs @ h)
 
 
-@deprecate_kwarg(kwarg_mapping={'neighbourhood': 'neighborhood'},
-                 removed_version="1.2",
-                 deprecated_version="0.19.2")
 def perimeter(image, neighborhood=4):
```

Comment:
Please have a look. I think `removed_version="1.2"` is actually supposed to be 1.0 which would apply to this release 0.20. It's a bit of unfortunate communication to the user.

Reply:
I think so too. Plus, I realize that with https://github.com/scikit-image/scikit-image/pull/6204, 'neighbourhood' wasn't simply deprecated in v0.19.2, but plain replaced with 'neighborhood' so, anyway, thanks for all this cleanup!